indigo/distribution.yaml

```diff
@@ -12684,6 +12684,16 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
```

Comment:
This needs to refer to a valid branch or tag name. Your repository has the "master" and "release" branches, but not an "indigo" branch. If you fix this, then I believe CI will be happy and we can merge.